features/proxy_config.feature

```diff
@@ -72,47 +84,54 @@ Feature: Proxy configuration
          \*   sudo ua config set apt_http_proxy=<value>
          \*   sudo ua config unset apt_http_proxy
```

Comment:
We need to update this message to something like:
```
        /\*
         \* Autogenerated by ubuntu-advantage-tools
         \* Do not edit this file directly
         \*
         \* To change what ubuntu-advantage-tools sets, use the `ua config set` command
         \* command to set either:
         \*     global_apt_http_proxy and global_apt_https_proxy
         \* for a global apt proxy.
         \* or
         \*     ua_apt_http_proxy and ua_apt_https_proxy
         \* for a apt proxy that only applies to UA related repos.
         \*\/
```

Reply:
I could you this as it is , I will just add the unset command option as well